tribe-ext-export-mec-events.php

```diff
@@ -24,6 +24,8 @@
  */
 
 // Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+use Tribe\Events\Admin\Settings;
+
```

Comment:
Does $this->add_required_plugin( 'Tribe__Events__Main' ); (below, line 41, in unedited code) require a TEC version set?